packages/pa_ppx_string/pa_ppx_string.0.01/opam

```diff
@@ -0,0 +1,39 @@
+synopsis: "A Camlp5 PPX Rewriter for String Interpolation "
+description:
+"""
+This is a PPX Rewriter for interpolating strings, meant to
+be a work-alike to ppx_string, but instead based on Camlp5
+(so it's compatible with all the other Camlp5-based PPX rewriters).
+"""
+opam-version: "2.0"
+maintainer: "Chet Murthy <chetsky@gmail.com>"
+authors: ["Chet Murthy"]
+homepage: "https://github.com/camlp5/pa_ppx_string"
+license: "BSD-3-Clause"
+bug-reports: "https://github.com/camlp5/pa_ppx_string/issues"
+dev-repo: "git+https://github.com/camlp5/pa_ppx_string.git"
+doc: "https://github.com/camlp5/pa_ppx_string/doc"
+
+depends: [
+  "ocaml"       { >= "4.10.0" & < "5.01.0" }
```

Comment:
```suggestion
  "ocaml"       { >= "4.10.0" & < "5.1.0" }
```
Is the upper bound on "5.1" really needed? We could add it later if something breaks

Reply:
Good point, removed the upper bound.